hw/dv/tools/dvsim/sim.mk

```
@@ -88,38 +83,46 @@ ifneq (${sw_images},)
 			echo "SW image \"$$bazel_label\" is prebuilt - copying sources."; \
 			cp ${proj_root}/$${prebuilt_path} $${run_dir}/`basename $${prebuilt_path}`; \
 		else \
-			echo "Building SW image \"$${bazel_label}\"."; \
-			bazel_opts="${sw_build_opts} --define DISABLE_VERILATOR_BUILD=true"; \
-			if [[ -z $${BAZEL_PYTHON_WHEELS_REPO} ]]; then \
-				echo "Building \"$${bazel_label}\" on network connected machine."; \
-				bazel_cmd="./bazelisk.sh"; \
-			else \
-				echo "Building \"$${bazel_label}\" on air-gapped machine."; \
-				bazel_opts+=" --distdir=$${BAZEL_DISTDIR} --repository_cache=$${BAZEL_CACHE}"; \
-				bazel_cmd="bazel"; \
-			fi; \
-			echo "Building with command: $${bazel_cmd} build $${bazel_opts} $${bazel_label}"; \
-			$${bazel_cmd} build $${bazel_opts} $${bazel_label}; \
-			for dep in $$($${bazel_cmd} cquery "labels(data, $${bazel_label})" \
+			target_labels+=$${bazel_target}+" "; \
```

I think you mean:
```suggestion
			target_labels+=$${bazel_label}+" "; \
```
`bazel_target` is the `bazel_label` after the colon, but without the device suffix (i.e., `_sim_dv`) attched.

---

nit: wdyt about naming it `bazel_labels` to be consistent with current naming scheme?